pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -123,10 +123,10 @@ func (admitter *VMICreateAdmitter) Admit(ar *admissionv1.AdmissionReview) *admis
 		return webhookutils.ToAdmissionResponse(causes)
 	}
 
-	reviewResponse := admissionv1.AdmissionResponse{}
-	reviewResponse.Allowed = true
-	reviewResponse.Warnings = append(reviewResponse.Warnings, warnDeprecatedAPIs(vmi)...)
-	return &reviewResponse
+	return &admissionv1.AdmissionResponse{
```

Comment:
This change appears in the `virt-api, vmi create admitter: Return undeclered AddmisstionResponse{}`.
I do not understand how the subject helps explain what was done here.
If I understand correctly, you just refactored this portion.

Reply:
Yeah this is a small refactoring, I though this title will explain more then just saying "refactor".